packs/outlaws-of-alkenstar-bestiary/anjelique-loveless.json

```
@@ -1137,4 +1141,4 @@
         }
     },
     "type": "npc"
-}
+}
```

```suggestion
}

```
Don't edit the json files directly, do it in Foundry then extract